src/redux/appReducer.js

```diff
@@ -0,0 +1,26 @@
+import * as buttonActions from '../pressButtonLogic';
+import * as types from './actionTypes';
+
+export default function appReducer(
+    state = {
+        result: ['0'],
+        operators: {
+            array: [],
+            redundant: 0,
```

Comment:
Please try to save in the state only the value that should be displayed and the array of buttons pressed. That should be enough to be able to process the result on each button press.